utils/vscode/syntaxes/carbon.tmLanguage.json

```
@@ -0,0 +1,178 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"name": "carbon",
+	"patterns": [
+		{
+			"include": "#comments"
+		},
+		{
+			"include": "#operators"
+		},
+		{
+			"include": "#strings"
+		},
+		{
+			"include": "#numbers"
+		},
+		{
+			"include": "#reserved-words"
+		},
+		{
+			"include": "#operator-dedicated-keywords-statements"
+		},
+		{
+			"include": "#ctrl-statements"
+		},
+		{
+			"include": "#special-keywords"
+		},
+		{
+			"include": "#true-false"
+		},
+		{
+			"include": "#functions"
+		},
+		{
+			"include": "#customs"
+		}
+	],
+	"repository": {
+		"operators": {
+			"patterns": [
+				{
+					"name": "keyword.operator.new.carbon",
+					"match": "\\b(\\+|-|\\*|/|!)\\b"
+				}
+			]
+		},
+		"comments": {
+			"patterns": [
+				{
+					"name": "comment.line.double-dash.carbon",
```

```suggestion
					"name": "comment.line.double-slash.carbon",
```

Also, in the plist version we have:

```json
    {
      "begin": "(^[ \\t]+)?(?=//)",
      "end": "(?!\\G)",
      "patterns": [
        {
          "name": "comment.line.carbon",
          "begin": "//",
          "end": "$"
        }
      ]
    },
```
... which I think is more correct: a `//` only starts a comment if it's at the start of a line.